### PR TITLE
Project Juno Moneta

### PR DESCRIPTION
**Pull request by Daniel-Szego**
Project Juno Moneta is an experimental blockchain based open-source CBDC platform for different wholesale and crossborder usecases.

### CODE REVIEW

File: labs/project-juno-moneta.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+https://github.com/Daniel-Szego
+https://github.com/hbalazs1980
+
+# Sponsor
+
+
+# Pre-existing repository
+
+https://github.com/Daniel-Szego/Project-Juno-Moneta
```
Comment (tkuhrt) [inline labs/project-juno-moneta.md:35]: It looks like some of the commits do not have proper sign-offs. Here is a guide on [how to sign previous commits](https://hyperledger-indy.readthedocs.io/projects/sdk/en/latest/docs/contributors/signing-commits.html#how-to-sign-previous-commits). Happy with this lab once that is done.

**Review by tkuhrt — APPROVED** [review]

File: labs/project-juno-moneta.md (hunk 1 above)
Comment (Daniel-Szego) [reply labs/project-juno-moneta.md:35]: Thanks for your feedback and effort, I have recreated the preexisting reporsitory, having one branch, one-commit, with a signoff from me.

**Review by knagware9 — APPROVED** [review]

**Review by lehors — CHANGES_REQUESTED** [review]
Please, remove the word "Project" from the name of your lab throughout. This term is reserved for Hyperledger Projects.
LGTM otherwise.
Thanks.

**Comment by ryjones** [comment]
Hi @Daniel-Szego, if you would like to move forward with this lab, please re-propose it after addressing the feedback from Lab Stewards.

**Pull request closed without merging**